generators/app/files.js

```diff
@@ -93,20 +90,8 @@ function writeFiles(generator) {
         return shelljs.cat(constants.MODULES_HOOK_FILE).match(constants.MODULE_NAME) === null;
     };
 
-    const generateKafkaProperties = (generator, enabled) => {
-        generator.enabled = enabled;
-
-        let generatedKafkaProperties = constants.EMPTY_STRING;
-        jhipsterUtils.renderContent(
-            generator.templatePath('src/main/resources/application-kafka.yml.ejs'),
-            generator,
-            generator,
-            {},
-            res => {
-                generatedKafkaProperties = res;
-            }
-        );
-        return generatedKafkaProperties;
+    const isCleanup = generator => {
+        return generator.props.cleanup ? generator.props.cleanup : false;
```

Comment:
No need for `isCleanup` you should simply use `generator.props.cleanup` in your conditions.